pcdscalc/diffraction.py

```diff
@@ -5,9 +5,12 @@
 require energy in eV.
 """
 import numpy as np
+import xraylib
```

Comment:
We need to add `xraylib` as a runtime dependency of `pcdscalc` in https://github.com/pcdshub/pcdscalc/blob/master/conda-recipe/meta.yaml and https://github.com/pcdshub/pcdscalc/blob/master/requirements.txt to import it here

Reply:
Almost there now. Only the pip 3.11 fails